department-of-veterans-affairs/va.gov-cms
# VACMS-8033: Update flag word-wrap style.

## Description
Closes #8033 .

## Testing done
viewed flags on smaller breakpoints and verified none wrapped.

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/154125944-06524ac1-242c-4342-aa1a-666334adc841.png)


## QA steps

As CMS admin, I can 
1. visit the flag view (`/admin/content/flagged`) and
2. verify the flags do not break and wrap to multiple lines 
3. Then validate Acceptance Criteria from issue
- [x] Flags should print on one line, without wrapping.

### Definition of Done


### Select Team for PR review

  - [x `⭐️ Content ops`


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
